src/modules/expenses/expense.repository.ts

```diff
@@ -0,0 +1,50 @@
+import type { TInsertExpenseSchema } from "@/db/schemas/expense.model";
+
+import { db } from "@/db/adapter";
+import expenseModel from "@/db/schemas/expense.model";
+
+interface TExpensePayloadUser {
+  amount: TInsertExpenseSchema["amount"];
```

Comment:
I believe using Pick would be better than doing this way, what do you think?

Reply:
I didn't find any as such difference yet. But `Pick` seems more useful utility. It reduced the code to one line.